src/features/BlockScreen.tsx

```diff
@@ -0,0 +1,64 @@
+import icon from "data-base64:~/assets/icon.png"
+import cssText from "data-text:~style.css"
+import type { PlasmoCSConfig, PlasmoGetInlineAnchor } from "plasmo"
+import { useEffect, useState } from "react"
+
+export const config: PlasmoCSConfig = {
+  matches: ["<all_urls>"],
+  all_frames: true,
+  css: ["font.css"]
+}
+
+export const getStyle = () => {
+  const style = document.createElement("style")
+  style.textContent = cssText
+  return style
+}
+
+export const getOverlayAnchor: PlasmoGetInlineAnchor = async () =>
+  document.querySelector(`body`)
+
+const BlockScreen = () => {
+  const [hide, setHide] = useState(false)
+
+  useEffect(() => {
+    //TODO every site is not going to be block by default, this is just an illux
+    document.body.style.overflowY = "hidden"
+  }, [])
+
+  const handleUnblock = () => {
+    document.body.style.overflowY = "scroll"
+    setHide(true)
+  }
+
+  if (hide) return null
+  return (
+    <>
+      <div className="w-screen h-screen p-12 text-black bg-white">
+        <div className="flex flex-col items-center justify-between w-full h-full">
+          <div></div>
+          <div className="flex space-x-2">
+            <img
+              onClick={() => handleUnblock()}
+              className="w-10 h-10 cursor-pointer"
+              src={icon}
+              alt="focus defender icon"
+            />
+            <h2 className="text-3xl font-bold">FocusDefender</h2>
```

Comment:
Since we're using this Logo in a couple of places. I'd suggest we create a component for it and then use the component in all those places instead of rebuilding it every time. 
It is supposed to be done in [this ticket](https://github.com/orgs/DotCampus/projects/5/views/1?pane=issue&itemId=41925948), but I don't think @cgnnanna has got around to it yet.

Reply:
Yeah sure, but however in some edges cases like here, where we had to listen to an on click, I don't think the components will be applicable here, however we can go ahead to create one, and use it anywhere it's stale